scss/tech.scss

```diff
@@ -1,63 +1,45 @@
-$tech-box-border-radius: 10px;
-$tech-box-height: 480px;
-$tech-box-width: 300px;
 $tech-logo-size: 100px;
-$tech-text-margin: 20px;
+$tech-text-margin: 10px;
 
 .tech-box-list {
-  display: flex;
-  flex-wrap: wrap;
-  margin: 0 0 50px;
-  padding: 0 50px;
+  align-items: start;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  margin: 0 auto 80px; 
+  width: 70%; //arbitrary value
```

Comment:
Dont use percentages, it may cause bad rendering, its better to use `px` values in this case